kin-sdk/src/main/java/kin/sdk/AccountStatus.java

```diff
@@ -18,12 +17,7 @@
      */
     int NOT_CREATED = 0;
     /**
-     * Account was created, but not activated yet, thus cannot send or receive kin yet. call {@link
-     * KinAccount#activate()} for activating.
-     */
-    int NOT_ACTIVATED = 1;
-    /**
-     * Account was created and activated, account is ready to use with kin.
+     * Account was created, account is ready to use with kin.
```

Comment:
I think we decided on 'Created' state (removing activate notion entirely). 
@Ronserruya that what you did in python, right?
@cnotethegr8 making sure you also aligned

Reply:
Changed to 'created' and also change method names and anything which includes activate